build_tools/cmake/iree_hal_cts_test_suite.cmake

```diff
@@ -116,6 +116,8 @@ function(iree_hal_cts_test_suite)
             "${IREE_ROOT_DIR}/iree/hal/cts/testdata/${_FILE_NAME}.mlir"
           FLAGS
             ${_TRANSLATE_FLAGS}
+          TRANSLATE_TOOL
+            "iree_tools_iree-translate"
```

Comment:
some CI failures may be fixed by just using "iree-translate" here

https://github.com/google/iree/blob/2f0a48cc909e96296f053974d515cff03b842665/build_tools/cmake/iree_macros.cmake#L105-L108

Reply:
Thanks - that worked.